src/c++/uds/src/uds/murmurhash3.c

```diff
@@ -8,7 +8,24 @@
 
 #include "murmurhash3.h"
 
+#ifndef VDO_UPSTREAM
+#include <linux/version.h>
+#undef VDO_USE_NEXT
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(10, 0))
+#define VDO_USE_NEXT
+#endif
+#else /* !RHEL_RELEASE_CODE */
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 11, 0))
```

Comment:
This version check is not right. This change actually shows up in 6.12-rc2. It should definitely not be used for any kernel version less than 6.12. (Hopefully we won't ever have to check the EXTRA_VERSION to support 6.12-rc1.)